### PR TITLE
V3

### PR DESCRIPTION
**Pull request by twelve1995**
### What problem does this PR solve? 
Comment submission.

### What is changed and how it works?
indicate why descscan needs to multiply by 2 when calculating overhead

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update the `tidb-ansible` repository
 - Need to be included in the release note

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tidb/8739)

### CODE REVIEW

File: planner/core/find_best_task.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ const (
 	netWorkFactor      = 1.5
 	netWorkStartFactor = 20.0
 	scanFactor         = 2.0
-	descScanFactor     = 2 * scanFactor
+	descScanFactor     = 2 * scanFactor //2 is the descscan requires scan twice
```
Comment (zz-jason) [inline planner/core/find_best_task.go]: > requires scan twice

No, it's not.

Comment (zz-jason) [reply planner/core/find_best_task.go]: @winoros @zhangjinpeng1987 PTAL

**Comment by shenli** [comment]
@1100hk The comment is not correct.

File: planner/core/find_best_task.go (hunk 1 above)
Comment (zhangjinpeng1987) [reply planner/core/find_best_task.go]: These factors here are used to estimate the cost of a physical plan. Here 2 means descend scan will be slower than ascend scan, descend scan will takes 2 times of the time compare to ascend scan for scan the same number of rows.

**Pull request closed without merging**